.gitignore

```diff
@@ -0,0 +1,3 @@
+./Debug/*
```

Comment:
Good stuff, apologies for adding this, I accidentally messed up and needed to restore from git and project configs were messed up, so i decided to add them, but i now realize that was a mistake